pipeline_dp/accumulator.py

```diff
@@ -30,22 +30,20 @@ def create_accumulator_params(
 ) -> typing.List[AccumulatorParams]:
     accumulator_params = []
     mechanism_type = aggregation_params.noise_kind.convert_to_mechanism_type()
+    budget_accountant = budget_accountant.request_budget(
```

Comment:
The current implementation is correct, we need a call to `request_budget` for each operations, it's how the API of `BudgetAccountant` works.

This PR makes that only 1 request is done as a result each metric get the full budget (i.e. budget exceeding), and so there is less noise.